Fantasy Battle Game/Assets/Scripts/Champions/Scripts/Champion.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+
+public class Champion : MonoBehaviour
+{
+    public float CurrentX { set; get; }
+    public float CurrentY { set; get; }
+    public float CurrentZ { set; get; }
+    public int CurrentHp { set; get; }
+    public bool IsFirstPlayer;
+    public int Damage;
+
+    public void SetPosition(float x, float y, float z)
```

Comment:
Add a vector of 3 option, to allow fast adding. -> {X,Y,Z}

You can aswell add variadic parameter here.